lib/brew_dash_web/live/brew_card_component.ex

```diff
@@ -6,10 +6,28 @@ defmodule BrewDashWeb.BrewCardComponent do
 
   @default_image "/images/default_brew.jpg"
 
-  def image_url(%Brew{recipe: %Recipe{image_url: nil}}),
+  def image_url(%Brew{recipe: recipe}) when recipe == nil or recipe.image_url == nil,
     do: Routes.static_path(BrewDashWeb.Endpoint, @default_image)
 
   def image_url(%Brew{recipe: %Recipe{image_url: image_url}}), do: image_url
 
-  def show_brew_name?(%Brew{recipe: recipe, name: name}), do: recipe.name != name
+  def display_name(%Brew{} = brew) do
```

Comment:
This works, but one thing it does is cut us off from the path of having different styles for different section of the display name. 
Eg:
**Folkol**-_Idaho-7_ (_**batch 3**_)

That is fine if we do not care to go down that path. Otherwise I would suggest moving display name into its own component.

Reply:
Yeah, I was thinking the same.  I was imagining this would get pulled out somewhere when we want to use it else where or have different displayable names.